app/helpers/application_helper/button/ems_infra_scale.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationHelper::Button::EmsInfraScale < ApplicationHelper::Button::Basic
+  def role_allows_feature?
+    super && role_allows?(:feature => "ems_infra_scale")
```

Comment:
@vecerek I dont think that this is necessary. We have product features for both `ems_infra_scale` and `ems_infra_scaledown` so the original `role_allows_feature?` in `basic.rb` should be enough.